sinch/domains/numbers/endpoints/callbacks/get_configuration.py

```diff
@@ -0,0 +1,28 @@
+from sinch.core.models.http_response import HTTPResponse
+from sinch.domains.numbers.endpoints.numbers_endpoint import NumbersEndpoint
+from sinch.core.enums import HTTPAuthentication, HTTPMethods
+
```

Comment:
grouping import, is blank line necessary?

Reply:
fixed